cmd/crio/config.go

```diff
@@ -172,6 +172,13 @@ default_capabilities = [
 default_sysctls = [
 {{ range $sysctl := .DefaultSysctls}}{{ printf "\t%q, \n" $sysctl}}{{ end }}]
 
+# List of additional devices. specified as
+# "<device-on-host>:<device-on-container>:<permissions>", for example: "--device=/dev/sdc:/dev/xvdc:rwm".
+#If it is empty or commented out, only the devices
+# defined in the container json file by the user/kube will be added.
+additional_devices = [
```

Comment:
I would call this `default_devices` so we're consistent with other options like default_ulimits and capabilities

Reply:
They are not default. These are additional.  The other cases they set the list of all capabiltiies and ulimits. In this case she is adding additional devices to the builtin ones like /dev/console, /dev/null, /dev/random ... I don't believe we want to list these.

I would stick with additional_devices.

Reply:
That makes sense, but then `default_mounts` should have been `additional_mounts` right? `default_sysctl` should become `additional_sysctl` as well, `default_ulimit` likewise actually, since it's just setting some of the ulimits but not all

Reply:
Well, for sysctl and ulimit, no, since the container engine (cri-o) does not set any ulimits or sysctl's by default.  So calling these additional versus default makes little difference.  If I show the default_ulimits=[] and default_sysctl=[] then the admin knows that CRI-O is not setting any ulimits or sysctls.  
Mounts might be a little tricky, since CRI-O does setup default mounts.  But I am not sure it is worth changing at this point.
I don't want to make the same mistake with devices.